code-gen-projects/java/code-gen-demo/src/test/java/org/example/CodeGenTest.java

```diff
@@ -39,7 +45,7 @@ class CodeGenTest {
          assertEquals("hi", s.getA(), "s.getA() should return \"hi\"");
          s.setB(6);
          assertEquals(6, s.getB(), "s.getB() should return `6`");
-         s.setC(new AnonymousType3(new ArrayList<String>()));
+         s.setC(new ArrayList<String>());
          assertEquals(true, s.getC().getValue().isEmpty(), "s.getC().isEmpty() should return `true`");
```

Comment:
I didn't change the getters because for an inline type that is for example a struct it would require to return the struct(e.g. check [this test case](https://github.com/amazon-ion/ion-cli/pull/109/files/f34a3fe433468b996ce6b741a0f6745b941db06f#diff-8fddb5eb93f33a051ff30f87fc97456d02b921ed0d46cb4b78bd889d7c33cb75R81) of nested struct hat has a field c which is also a struct ). This will be simplified once we have `$codegen_name` or similar which will return class `C` instead of `AnonymousTypeX`.
Other option is to instead let all the `getX()` methods of the nested struct be part of the parent class.